testutils/src/main/scala/za/co/absa/CmdConfig.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.standardization
+
+import scopt.OptionParser
+
+/**
+  * This is a class for configuration provided by the command line parameters
+  *
+  * Note: scope requires all fields to have default values.
```

Comment:
```suggestion
  * Note: scopt requires all fields to have default values.
```
I thought you'd copy it from standardization or confromance.